meta-intel-edison-distro/recipes-core/images/core-image-minimal-initramfs.bbappend

```diff
@@ -9,8 +9,8 @@ PACKAGE_INSTALL += " kernel-module-mmc-block"
 PACKAGE_INSTALL += " kernel-module-sdhci-acpi"
 PACKAGE_INSTALL += " kernel-module-sdhci-pci"
 PACKAGE_INSTALL += " ${@bb.utils.contains('DISTRO_FEATURES', 'acpi', 'acpi-tables', '', d)}"
-PACKAGE_INSTALL_remove = " initramfs-live-install"
-PACKAGE_INSTALL_remove = " initramfs-live-install-efi"
+PACKAGE_INSTALL_remove = " initramfs-live-install initramfs-module-install"
```

Comment:
Let's add `initramfs-module-setup-live` as well, unless you see a reason to preserve it.

Reply:
I just tried removing, but then we don't have boot. The reason is that removing this also removes `udev-extraconf` which apparently holds rules for udev.